app/renderer/js/pages/preference/general-section.ts

```diff
@@ -104,7 +104,6 @@ class GeneralSection extends BaseSection {
 				<button class="custom-css-button green">${t.__('Upload')}</button>
 			</div>
 			<div class="setting-row" id="remove-custom-css">
-		"S
```

Comment:
This was accidentally included by @kanishk98 in the translation PR 😄!

Reply:
Nice catch :)